qiskit/providers/aer/pulse/controllers/pulse_controller.py

```diff
@@ -422,7 +423,15 @@ def init_rhs(self, exp):
         # Init register
         register = np.ones(self.n_registers, dtype=np.uint8)
 
-        ode_rhs_obj = get_ode_rhs_functor(self._rhs_dict, exp, self.system, channels, register)
+        rhs_dict = self._rhs_dict
+
+        # if the experiment overrides qubit_lo_freq, use this one
+        if exp.get('qubit_lo_freq', None) is not None:
```

Comment:
why not just `if 'qubit_lo_freq' in exp:`?

Reply:
True :)

Reply:
Actually I'm still going to keep a check for whether it `is not None`